modules/client/src/main/java/org/apache/ignite/internal/client/tx/DirectTxUtils.java

```diff
@@ -131,6 +133,10 @@ public static void writeTx(@Nullable Transaction tx, PayloadOutputChannel out, @
                     out.out().packLong(tx0.observableTimestamp());
                     out.out().packBoolean(tx.isReadOnly());
                     out.out().packLong(tx0.timeout());
+                    if (ctx.channel.protocolContext().isFeatureSupported(TX_CLIENT_GETALL_SUPPORTS_PRIORITY)
```

Comment:
Is there a chance that we need `lowPriority` flag for other operations in the future? We could pass it to all operations and use a more general name for the feature flag. Maybe even replace a boolean with a flags varint so it is easier to extend later.